bin/cdist

```diff
@@ -44,6 +43,43 @@ import cdist.shell      # noqa 402
 import cdist.inventory  # noqa 402
 
 
+py_vers = sys.version_info[:3]
+
+
+def cdist_logger():
+    import cdist.log
+    import logging
+    return logging.getLogger("cdist")
```

Comment:
why wrapping this into function?

Reply:
I initially thought that it would be cleaner to only initialise the logging facility when an error occurs.
But after giving it some more thought I believe that the idea was flawed, because if an error occurs early in the initialisation process it might as well be in the `cdist.log` module, breaking this whole approach.

I changed the code to only use the `logging` library (which is available on Python 2.3+).
So it will use coloured logging, etc. if the logging facility had already been set up or otherwise just fall back to the default.